js/textbox/shared/Textbox.svelte

```diff
@@ -270,7 +270,7 @@
 					dir={rtl ? "rtl" : "ltr"}
 					bind:value
 					bind:this={el}
-					{placeholder}
+					placeholder={disabled ? "" : placeholder}
```

Comment:
Unrelated fix but the textbox component was showing a "Type here..." placeholder even when it was disabled. This removes that.